.github/workflows/wheel.yaml

```diff
@@ -4,6 +4,9 @@ on:
   push:
   pull_request:
   workflow_dispatch:
```

Comment:
We'd probably ... not want these triggers, right?

Reply:
maybe PR, but not push and workflow_dispatch... They won't upload unless it's a tagged version anyways.

Reply:
Do we want the `cron`? It says its set up for every Monday.

Reply:
I didn't notice before that the last step (uploading to PyPI) only happens when a release is published. So I suppose running every week seems fine :)